doc/news/changes/minor/20200128PasqualeClaudioAfrica

```diff
@@ -1,4 +1,3 @@
-Improved: ParameterHandler class can now optionally print parameters in the same
-order as they are declared, rather than sorted in alphabetical order.
```

Comment:
It looks like you are mixing up your old changelog. Shouldn't the new change go into 20200129*?